datalad/utils.py

```diff
@@ -1088,6 +1088,20 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.__class__(self._prev_pwd, logsuffix="(coming back)")
 
 
+def dlabspath(path, norm=False):
+    """Symlinks-in-the-cwd aware abspath
+
+    os.path.abspath relies on os.getcwd() with would not know about symlinks
```

Comment:
s/with/which/ (tried to just push the fixup, but permission was denied.  perhaps something to do with the issue->pr conversion?)

Reply:
done, thanks!

indeed, it might be that with issue->PR by default it makes it unmarked... something to check how to change ;-)